ui/raidboss/data/04-sb/dungeon/bardams_mettle.js

```diff
@@ -0,0 +1,315 @@
+      netRegexJa: NetRegexes.startsUsing({ id: '1EFA', source: 'ガルラ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1EFA', source: '加鲁拉', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      // Both Bardam and Yol use the 0017 head marker.
+      // If we're in the Yol encounter, we're obviously not fighting Bardam.
+      id: 'Bardam\'s Mettle Dead Bardam',
+      netRegex: NetRegexes.message({ line: '.*Voiceless Muse will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.deadBardam = true;
+      },
+    },
+    {
+      id: 'Bardam\'s Mettle Empty Gaze',
+      netRegex: NetRegexes.startsUsing({ id: '1F04', source: 'Hunter Of Bardam', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1F04', source: 'Bardams Jäger', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1F04', source: 'chasseur de Bardam', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1F04', source: 'バルダムズ・ハンター', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1F04', source: '巴儿达木的猎人', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Bardam\'s Mettle Sacrifice',
+      netRegex: NetRegexes.startsUsing({ id: '1F01', source: 'Bardam', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1F01', source: 'Bardams Statue', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1F01', source: 'Bardam', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1F01', source: 'バルダムの巨像', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1F01', source: '巴儿达木巨像', capture: false }),
+      infoText: {
+        en: 'Stand in a tower',
+        de: 'Im Turm stehen',
+        fr: 'Tenez-vous dans une tour',
+        ko: '장판 들어가기',
+      },
+    },
+    {
+      // Bardam casts Comet repeatedly during this phase,
+      // but 257D is used only once. The others are 257E.
+      id: 'Bardam\'s Mettle Comet',
+      netRegex: NetRegexes.startsUsing({ id: '257D', source: 'Bardam', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '257D', source: 'Bardams Statue', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '257D', source: 'Bardam', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '257D', source: 'バルダムの巨像', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '257D', source: '巴儿达木巨像', capture: false }),
+      alertText: {
+        en: '8x puddles on YOU',
+        de: '8x Fläche auf DIR',
+        fr: '8x Zones au sol sur vous',
+      },
+    },
+    {
+      id: 'Bardam\'s Mettle Meteor Impact',
+      netRegex: NetRegexes.startsUsing({ id: '2582', source: 'Looming Shadow' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2582', source: 'Lauernd[a] Schatten' }),
```

Comment:
Same here again, please replace the `[a]`. it should be `Lauernd(?:e|er|es|en) Schatten`